book/website/collaboration/research-infrastructure-roles/sustain-rirs.md

```diff
@@ -0,0 +1,107 @@
+<!-- 
+Identify the major themes/topics that make up your chapter. 
+These will be the subchapters of your chapter.
+Write about each one in a different `chapter-content.md` file.
+Make as many copies of this file as you need for all your subchapters
+-->
+
+(cl-infrastructure-sustain-rirs)=
+# Sustaining Research Infrastructure Roles
+
+> **Write an introduction for your subchapter here**.
+> Your introduction should briefly introduce the topic of your subchapter and highlight its key ideas.
+> Even though your introduction should be short and focused, you should try to explain why and how the subchapter fits the overall context of the chapter and what the reader should expect to learn from it.
+
+<!-- 
+> Each of the key ideas you talk about in the introduction should have a section of its own.
+> When you mention a key idea in your introduction, remember to cross reference it to the section of your subchapter where you explain it further.
+> This will be useful for people who may only want to read specific parts of your content.
+> See the [style guide](https://the-turing-way.netlify.app/community-handbook/style/style-crossref.html) for The Turing Way's recommendations on cross referencing.
+
+> In this template, we assume that our subchapter has three key ideas, your subchapter may have more or less than this. 
+
+> It may be beneficial to run your content through a grammar checker (such as Grammarly) to catch grammatical mistakes.
+-->
+
+<!-- 
+In the label, replace `keyidea1` with a word that best describes the section or key idea you want to explain -->
+(cl-infrastructure-sustain-rirs-keyidea1)=
+## Key Idea 1
+
+> **Talk about your key idea in detail**. Feel free to use images, code blocks, and admonitions to communicate your ideas.
+> You may break the section down into subsections if you wish, however, remember to add [labels](https://the-turing-way.netlify.app/community-handbook/style/style-crossref.html) to any additional headers you create to facilitate cross-referencing.
+
+> Remember to use the [style guide](https://the-turing-way.netlify.app/community-handbook/style.html) and Jupyter Book's [Cheat Sheet](https://jupyterbook.org/reference/cheatsheet.html) to guide your writing.
+> The [style guide](https://the-turing-way.netlify.app/community-handbook/style/style-citing.html) also contains _The Turing Way's_ recommendations for referencing and citation.
+
+> To include an image in your writing, use the MyST directive shown below. 
+> Remember to add your image to the `figures` [folder](https://github.com/alan-turing-institute/the-turing-way/tree/main/book/website/figures) and use the correct path, else it will not be displayed.
+
+```{figure} ../../figures/image-name.png
+---
+name: image-name
+alt: describe your image for readers who rely on screen readers
+---
+Your image caption here
+```
+
+> To include code blocks, simply enclose your code in three sets of backticks shown below.
```

Comment:
We could worry about this formatting on our end, but maybe this doesn't need to be in the pads for ease of access/writing? What do you think?